recipe/bld.bat

```diff
@@ -1,8 +1,12 @@
 mkdir build && cd build
 
-cmake .. -G "NMake Makefiles"                  ^
-	-DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%    ^
-	-DCMAKE_BUILD_TYPE=Release                 ^
-	-DBUILD_SHARED_LIBS=OFF
+cmake .. -G "%CMAKE_GENERATOR%" ^
+	-DBUILD_SHARED_LIBS=ON ^
+	-DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+	-DENABLE_ZLIB_COMPRESSION=ON ^
+	-DCMAKE_BUILD_TYPE=Release ^
+	-DENABLE_DEBUG_LOGGING=ON
```

Comment:
I am not sure if this has anything to do with verbosity of logging. It helps in debugging the program linking to the library: http://trac.libssh2.org/wiki/FAQ#HowcanIdebugmylibssh2application . Should I turn it off then?